VisionDemo/procedures/tfExample.vail

```diff
@@ -0,0 +1,6 @@
+PROCEDURE tfExample(imageName String, modelName String, threshold Real)
+
+var results = TensorFlowOperation.processDocument(imageName, modelName, threshold)
+
+return results
```

Comment:
Why not just call TensorFlowOperatipon.processDocument directly from the client and remove this procedure entirely? It's better to direct users to built-ins when possible

Reply:
I tried calling it from the client and it is undefined so I think that the operation needs to be called from a procedure